README.md

```diff
@@ -1,183 +1,86 @@
-
-## How do I use it?
-
-### Creating your Team Repository
-
-One person in the team, preferably the leader, will have to create the repository and add other members as collaborators.
-
-1. In the top right corner of your screen, where **Clone** usually is, you have a **Use this template** button to click.
+### Media location
 
-![](https://docs.github.com/assets/images/help/repository/use-this-template-button.png)
-
-2. Give the repository a name and a description.
-
-![](https://docs.github.com/assets/images/help/repository/create-repository-name.png)
-
-3. Click **Create repository from template**.
-
-4. Click **Settings** in your newly created repository.
-
-![](https://docs.github.com/assets/images/help/repository/repo-actions-settings.png)
-
-5. Select **Manage access**.
-
-<!-- Yes, this is inline html. The source image is too vertical to be displayed with 100% width. -->
-<img src="https://docs.github.com/assets/images/help/repository/manage-access-tab.png" style="width: 30%"></img>
-
-6. Click **Invite a collaborator**.
-
-![](https://docs.github.com/assets/images/help/repository/invite-a-collaborator-button.png)
-
-7. Insert the names of each of your teammates, and invite them. Once they have accepted the invitation in their email, they will have write access to the repository.
-
-You are now ready to go! Now sit down, relax, and wait for the kickstart!
-Don't forget to swap "Python Discord" in the `LICENSE` file for the name of each of your team members or the name of your team after the start of the jam.
+- macOS and Linux
+```shell
+ls ~/skunkbooth/pictures
+```
 
-### Using the Default Pip Setup
+- Windows
+```shell
+dir C:/Users/<username>/skunkbooth/pictures
+```
 
-Our default setup includes a bare requirement file to be used with a [virtual environment](https://docs.python.org/3/library/venv.html).
+# Contributing
+[Poetry](https://python-poetry.org/) is used for package management.
 
-We recommend this if you never have used any other dependency manager, although if you have, feel free to switch to it. More on that below.
+### Install Poetry
 
-#### Creating the environment
-Create a virtual environment in the folder `.venv`.
+- macOS, Linux or WSL
 ```shell
-$ python -m venv .venv
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
 ```
 
-#### Enter the environment
-It will change based on your operating system and shell.
+- Windows Powershell
 ```shell
-# Linux, Bash
-$ source .venv/bin/activate
-# Linux, Fish
-$ source .venv/bin/activate.fish
-# Linux, Csh
-$ source .venv/bin/activate.csh
-# Linux, PowerShell Core
-$ .venv/bin/Activate.ps1
-# Windows, cmd.exe
-> .venv\Scripts\activate.bat
-# Windows, PowerShell
-> .venv\Scripts\Activate.ps1
+(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
```

Comment:
Same thing earlier with the installation.

Reply:
done